src/compiler/objective_c_plugin.cc

```diff
@@ -74,6 +80,24 @@ class ObjectiveCGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
       return true;
     }
 
+    ::grpc::string framework;
+    std::vector<::grpc::string> params_list =
+        grpc_generator::tokenize(parameter, ",");
+    for (auto param_str = params_list.begin(); param_str != params_list.end();
+         ++param_str) {
+      std::vector<::grpc::string> param =
+          grpc_generator::tokenize(*param_str, "=");
+      if (param[0] == "generate_for_named_framework") {
+        if (param[1].empty()) {
```

Comment:
I wonder if it will be index out of bound if someone writes `--grpc_out=generate_for_named_framework:.`. I don't think `tokenize` adds an empty element as the second parameter in this case.